### PR TITLE
adding dbdata

### PR DESCRIPTION
**Pull request by rmarchandTen24**
This change is [Reviewable](https://reviewable.io/reviews/ten24/slatwall/7060)

### CODE REVIEW

File: config/dbdata/SlatwallCollection.xml.cfm [hunk 1]
```diff
@@ -29,5 +29,6 @@
...
...gration ID","propertyIdentifier":"_entityqueue_integration.integrationID","isVisible":true,"isDeletable":true,"isSearchable":true,"isExportable":true,"ormtype":"string","type":"none","sorting":{"active":false,"sortOrder":"asc","priority":0},"key":"integration_integrationID"},{"title":"Integration | Integration Package","propertyIdentifier":"_entityqueue_integration.integrationPackage","isVisible":true,"isDeletable":true,"isSearchable":true,"isExportable":true,"ormtype":"string","type":"none","sorting":{"active":false,"sortOrder":"asc","priority":0},"key":"integration_integrationPackage"},{"title":",entity.entityqueue.trycount_en_us_missing,entity.entityqueue.trycount_en_missing,entity.define.trycount_en_us_missing,entity.define.trycount_en_missing,define.trycount_en_us_missing,define.trycount_en_missing","propertyIdentifier":"_entityqueue.tryCount","isVisible":true,"isDeletable":true,"isSearchable":true,"isExportable":true,"ormtype":"integer","type":"none","sorting":{"active":false,"sortOrder":"asc","priority":0}}],"keywordColumns":[],"filterGroups":[{"filterGroup":[]}],"currentPage":1,"pageShow":10,"defaultColumns":false,"dirtyRead":false,"periodInterval":""}' reportFlag='false'/>
```
Comment (migueltarga) [inline config/dbdata/SlatwallCollection.xml.cfm]: We are not using collections on workflows anymore, now the trigger has he config

Comment (rmarchandTen24) [reply config/dbdata/SlatwallCollection.xml.cfm]: @migueltarga tried switching it but we don't have the support for schedulecollectionconfig in core yet either.

Comment (migueltarga) [reply config/dbdata/SlatwallCollection.xml.cfm]: https://github.com/ten24/slatwall/blob/develop/model/entity/WorkflowTrigger.cfc#L55

Comment (rmarchandTen24) [reply config/dbdata/SlatwallCollection.xml.cfm]: @migueltarga yes the property is there but nothing that uses the property

**Pull request merged**